pkg/crc/constants/constants.go

```diff
@@ -21,8 +21,6 @@ const (
 
 	CrcEnvPrefix = "CRC"
 
-	DefaultBundleExtension = ".crcbundle"
-
```

Comment:
This should be moved to `Add GetBundleNameWithoutExtension function `

Reply:
I already removed it and have it on original location, like currently on master.

Reply:
Looking at the individual commit diffs, it's added to `pkg/crc/constants/constants.go` in `Add GetBundleNameWithoutExtension function` and then removed in the `Add generate command` commit, is this intended?

Reply:
This is still being moved to `pkg/crc/constants/constants.go` in the first commit, and then moved back in one of the later ones.